docs/simulating_with_the_mvs.rst

```diff
@@ -0,0 +1,200 @@
+=======================
+Simulating with the MVS
+=======================
+
+In the MVS can perform a capacity as well as dispatch optimisation of a specific energy system.
+This means that both the capacity that is to be bought is optimised as well as the respective assets operation.
+To perform an energy system simulation, a multitude of input parameters needed, which are described in link.
+They include economic parameters, technological parameters and project settings.
+Together they define all aspects of the energy system to the simulated and optimised.
+With this, the MVS builds an energy system model which is translated to an equation system which is to be solved.
+The MVS tries to minimise the annual costs of demand supply.
+
+In this section, we want to provide you with all information needed to run your own optimisations.
+First we will explain the two input types, json and csv,
+and then how to make an energy system model out of your real local energy system configuration.
+
+Input files
+-----------
+
+All input files stored in a specifically formatted input folder.
+The path to the input folder is up to you.
+
+.. image:: images/folder_structure_inputs.png
+ :width: 200
+
+There are two options to insert all input data – Json and CSV. These will be explained below.
+
+Json file: mvs_config.json
+##########################
+
+In combination especially with the planned graphical user interface for the MVS (EPA),
+a json file will be used to store and provide all input data necessary to understand relation.
+The Json file itself is created by the EPA, ie. there are no manual changes.
+You can use a specific Json input file if you want to test a simulation that has been made public online,
+one test simulation, or as a developer that has knowingly edited the Json file.
+
+As this requires to adhere to quite specific formatting rules,
+this can really only be recommended for advanced users.
+
+There can only be a single Json file in your input folder.
+As some parameters in the Json file link to a time series provided as a CSV,
+the folder "time_series" should be present in your input folder
+and provide all necessary input data that is an timeseries format.
+This can include for example PV generation time series demand time series.
+
+Csv files: csv_elements folder
```

Comment:
I would start with explaining csv and then json, as users, that are not developers can find their way faster.

Reply:
True, will change that next.